client/wc-api/settings/selectors.js

```diff
@@ -36,7 +36,23 @@ const isGetSettingsRequesting = getResource => group => {
 		return true;
 	}
 
-	return lastRequested > lastReceived;
+	// This selector is used by other "groups", so return early if not looking at wc_admin settings.
+	if ( 'wc_admin' !== group ) {
+		return lastRequested > lastReceived;
+	}
+
+	// Mutation operations for `wc_admin` settings update a different resource (batch endpoint) in fresh-data.
+	// As such we must use lastReceived stamp from that resource to properly compare lastRequested to lastReceived.
+	const { lastReceived: lastMutationReceived } = getResource(
+		getResourceName( 'settings/wc_admin', 'woocommerce_actionable_order_statuses' )
```

Comment:
What do you think about using the `analyticsSettings` object to pick a resource name here since the value is filterable? See: https://github.com/woocommerce/woocommerce-admin/blob/b0c2b15a208baebda17c091ab47c874c108c42c9/client/analytics/settings/config.js#L53

Reply:
@jeffstieler so just grab the first key from `analyticsSettings` instead of having the hardcoded one?

Reply:
Yeah, that's what I was thinking.